services/cloudfront/src/main/java/software/amazon/awssdk/services/cloudfront/CloudFrontUtilities.java

```diff
@@ -67,6 +67,8 @@
 @SdkPublicApi
 public final class CloudFrontUtilities {
```

Comment:
Missing the SigningUtils class.

Reply:
created class and moved previously private methods from CloudFrontUtilities